nftables_test.go

```diff
@@ -32,6 +32,8 @@ func newTestInterface(t *testing.T, family Family, tableName string) (Interface,
 	ip := "ip"
 	if family == IPv6Family {
 		ip = "ip6"
+	} else if family == InetFamily {
+		ip = "inet"
 	}
```

Comment:
this is goofy... I don't know why I did it that way. It should just use `family` directly below instead of `ip`

Reply:
done